packages/components/inputs/search-text-input/src/search-text-input.styles.ts

```diff
@@ -53,6 +54,9 @@ const getSearchTextInputStyles = (props: TInputProps) => [
     &:hover {
       background-color: transparent !important;
     }
+    font-size: ${props.isCondensed
+      ? `${designTokens.fontSize20}`
+      : `${designTokens.fontSize30}`};
```

Comment:
I am not so sure of the full context of this PR on why do we need `isCondensed` for just the SearchTextInput?

Also, Maybe we can generate a token for this state
```suggestion
      ? `${designTokens.fontSizeForInputWhenCondensed}`
      : `${designTokens.fontSize30}`};
```

So at least if this change is going to be made across other input components later on, then we have a variable for it. @FilPob what do you think ?